examples/plot_hf_hub.py

```diff
@@ -21,6 +21,7 @@
 
 import sklearn
 from huggingface_hub import HfApi
+import gradio as gr
```

Comment:
I think we should have a separate example for this.

Also need to add the dependency for doc build.

Note that this might actually break more often, since it might take a little while before gradio adapts to the latest skops. Adding this here means we expect gradio to work with the dev version of skops.